lib/stytch/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Stytch
-  VERSION = '10.17.0'
+  VERSION = '10.18.0'
```

Comment:
You can/should make this 10.17.1
```suggestion
  VERSION = '10.17.1'
```